app/config.js

```diff
@@ -29,6 +29,7 @@ config.apps = [
 config.auth0 = {
   domain: process.env.AUTH0_DOMAIN,
   clientID: process.env.AUTH0_CLIENT_ID,
+  clientId: process.env.AUTH0_CLIENT_ID,
```

Comment:
Why do you have both `clientID` and `clientId` with the same value here? I'd just stick with `clientID`.

Reply:
Different modules require different capitalisation :(

I'll change it to be more explicit about why